src/ColoredSort.java

```diff
@@ -0,0 +1,62 @@
+/***
+ *  @author Nicholas Maltbie
+ *
+ *  A colored sort adds a sall colored bar to the bottom of a sort that can
+ *  be used to show information about the sort.
+ */
+
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.FontMetrics;
+import java.awt.Graphics2D;
+import java.awt.RenderingHints;
+
+abstract public class ColoredSort extends Sort {
+
+    /**
+     * Default color of a colored marker.
+     */
+    public static final Color DEFAULT_COLOR = Color.GREEN;
+
+    /**
+     * Colors of the different indicies
+     */
+    private Color[] colors;
```

Comment:
Lets refactor this variable, and its getters and setters to something more telling before merging into ```Sort.java```. How about ```statusBarColors``` or something similar?